api/app_analytics/models.py

```diff
@@ -73,11 +73,14 @@ def check_overlapping_buckets(self):
 
 
 class FeatureEvaluationRaw(models.Model):
-    feature_name = models.CharField(max_length=2000)
+    feature_name = models.CharField(db_index=True, max_length=2000)
     environment_id = models.PositiveIntegerField()
     evaluation_count = models.IntegerField(default=0)
     created_at = models.DateTimeField(auto_now_add=True)
 
+    # Identity identifier stored for tracking multivariate split testing.
+    identifier = models.CharField(max_length=2000, null=True, default=None)
```

Comment:
Oh yeah, 100% good catch. I don't know why I didn't think of it.

Reply:
FYI: I'm updating a lot of the code to match this behaviour since there are a lot of places that refer to this only as `identifier` when `identity_identifier` would be more sound. It's taking some time but it's the right call.